### ISSUE TITLE
`GET /education/gi-bill/post-9-11/ch-33-benefit` erroneously shows the site is down for maintenance

### ISSUE THREAD

**Issue by lamawithonel** [department-of-veterans-affairs/vets-website#8966]
Prior to approval for benefits, the "Check Post-9/11 GI Benefits" link showed _always_ this message:

> **The Post-9/11 GI Bill Benefits tool is down for maintenance right now**
> 
> You can use the Post-9/11 GI Bill Benefits tool Sunday through Friday, 6:00 a.m. to 10:00 p.m. (ET), and Saturday 6:00 a.m. to 7:00 p.m. (ET). We do regular maintenance on the tool outside of these hours, and during that time you won't be able to use it.

It also shows this if I am not logged in when accessed from the `preview.va.gov` site.

At first I thought maybe it was truly a months-long period of down-time.  Calling the help line was helpful for the purposes of my benefits, but they they knew nothing of the web site-- **and it wastes a solid 30 minutes of my day waiting on hold!**  Since I've been approved for benefits, though, I now think the error message is simply wrong.

Since I've been approved, when I am logged in, it correctly shows my benefits status.  Still though, when I am logged out, it shows the "down for maintenance" message.

It would be more helpful if it correctly displayed that I had no records on file, that I needed to log in first, or whatever the underlying reason for the NULL return really is.  Always showing the "down for maintenance" message and asking that I try again later is confusing and wastes my time.
